src/cbor.zig

```diff
@@ -338,6 +380,15 @@ fn burn(data: []const u8, i: *usize) ?void {
                 }
             }
         },
+        0x9f => {
+            i.* += offset;
+            while (data[i.*] != 0xff) {
+                if (burn(data, i) == null) {
+                    return null;
+                }
+            }
+            i.* += 1;
+        },
```

Comment:
As burn is implemented recursively, this loop should always hit the corresponding break point. LGTM